autosklearn/estimators.py

```diff
@@ -52,6 +54,7 @@ def __init__(
         metric=None,
         scoring_functions: Optional[List[Scorer]] = None,
         load_models: bool = True,
+        get_trials_callback: Optional[IncorporateRunResultCallback] = None
```

Comment:
Could you please add the callback to the docstring so that it's rendered in the docs?

Reply:
Modified the code to add docstring.